apps/api/src/app/environments/usecases/create-environment/create-environment.e2e.ts

```diff
@@ -35,4 +35,20 @@ describe('Create Environment - /environments (POST)', async () => {
 
     expect(body.message[0]).to.contain('name should not be null');
   });
+
+  it('should create a default layout for environment', async function () {
+    const demoEnvironment = {
+      name: 'Hello App',
+    };
+    const { body } = await session.testAgent.post('/v1/environments').send(demoEnvironment).expect(201);
+
+    const layouts = await layoutRepository.find({
+      _environmentId: body.data._id,
+      channel: 'email',
+    });
```

Comment:
I usually prefer not use endpoints like this in e2e tests from the thinking that I'm testing the create environment functionality and not the `GET /api/layouts`, in some cases, it's even helped to surface bugs that crippled from the other endpoints. In this particular scenario, I'm making a very simple query to double-check that not duplicate entries were created and etc. 

Wdyt?

Reply:
I am on the other side and try as much E2E as possible, even for the checks. Here you are just using the `layoutRepository` to check that the data created is right, not a step to contribute to creation flow.
Anyway, this was just a suggestion. 🙂

Reply:
You have convinced me ;) 

Should we even use the @novu/node package for that? Trying to think if we can somehow create some reusable methods for this instead of just using hardcoded and repeated session.call methods

Reply:
It is a possibility. So far that endpoint is not used that much, but if it is in the future we could always add it to the testing package, as we do for other heavily used functionalities. Something to consider, indeed.